components/auth/src/main/java/org/cloudfoundry/credhub/config/auth/OAuth2AuthenticationExceptionHandler.java

```diff
@@ -36,6 +36,7 @@ public void commence(HttpServletRequest request,
             jsonObject.addProperty("error", error.getErrorCode());
             jsonObject.addProperty("error_description", error.getDescription());
         } else if (authException instanceof InsufficientAuthenticationException) {
+            jsonObject.addProperty("error", "invalid_token");
```

Comment:
So, this change is to satisfy the expectation of the test cases in `credhub_acceptance_tests`, correct?
I do not think  `invalid_token` is proper value for `InsufficientAuthenticationException`, though. How about we set it to "access_denied" (picked from https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1) and change the test expectations instead?

Reply:
Yes, this change was attempting to match the credhub_acceptance_tests exactly.  I have now added a commit here that uses the standard OAuth2 Error Response.